papers/scipy/scipy.rst

```diff
@@ -26,36 +18,56 @@
     same size as and side-by-side with the z.  Using `a_{_N}` makes it
     very clear that N is a subscript of a.
 
------
-SciPy
------
+-------------------------------------
+Discrete-time Linear Filters in SciPy
```

Comment:
As a title this sounds a bit technical, maybe something like "Signal processing with SciPy - linear filters"?

Reply:
Agreed, "signal processing" should be in the title.  I switch the hyphen to a colon: "Signal processing with SciPy: linear filters".